demos/JWST/S3_miri_lrs_template.ecf

```diff
@@ -20,6 +20,7 @@ p3thresh        5           # X-sigma threshold for outlier rejection during bac
 save_bgsub      False       # Whether or not to save background subtracted FITS files
 
 # Spectral extraction parameters
+record_ypos     False        # Option to record the y position and width for each integration (only records if src_pos_type is gaussian)
```

Comment:
Please document this new ECF setting at docs/source/ecf.rst

Reply:
done